solutions/usaco-944.mdx

```diff
@@ -100,3 +100,86 @@ public class FencePlan {
 </JavaSection>
   
 </LanguageSection>
+
+<LanguageSection>
+	
+<CPPSection>
+	
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+#define endl '\n'
+typedef long long ll;
+typedef pair<int, int> pii;
+
+struct Cow {
+    int x;
```

Comment:
tabs

Reply:
yeah it's still a problem lol

Reply:
idk how to fix it then, cause when I go to edit it in github it says that tabs are selected for indentation instead of spaces

Reply:
oh the indentation size was set to 8 before, so I set it to 4 now. Hopefully that works?